src/Microsoft.Fast.Components.FluentUI/Components/FluentAnchoredRegion.razor.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections.Generic;
+
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.Fast.Components.FluentUI;
+public partial class FluentAnchoredRegion
+{
+    /// <summary>
+    /// The html id of the HTMLElement used as the anchor around which the positioning region is placed. This must be set for the component's positioning logic to be active.
+    /// </summary>
+    [Parameter]
+    public string? Anchor { get; set; }
+
+    /// <summary>
+    /// The ID of the HTMLElement to be used as the viewport used to determine available layout space around the anchor element. If unset the parent element of the anchored region is used.
+    /// </summary>
+    [Parameter]
+    public string? Viewport { get; set; }
+
+    /// <summary>
+    /// Default is 'uncontrolled'.
+    /// </summary>
+    [Parameter]
+    public Positioning? HorizontalPositioningMode { get; set; } = Positioning.Uncontrolled;
+    /// <summary>
```

Comment:
Can we keep one horizonal space between properties?

Reply:
Yes, we can do that